sdk/eventhub/Azure.Messaging.EventHubs/design/proposal-custom-endpoint-address.md

```diff
@@ -0,0 +1,69 @@
+# Event Hubs: Custom Endpoint Support
+
+Connections to the Azure Event Hubs service are made using the fully qualified namespace name assigned for the Event Hubs namespace as the root of the endpoint address.  Because the Event Hubs service ensures the appropriate CNAME records are configured in DNS as part of the provisioning process, this provides an endpoint address that is strongly associated with the Event Hubs namespace and easier for developers to remember.
+
+While the standard approach works well under normal circumstances, it can be troublesome in certain environments when a proxy is in use or when an Express Route circuit is used with certain configuration.  In these scenarios, an alternative host name is needed to allow for proper routing to the Event Hubs service.
```

Comment:
```suggestion
While the standard approach works well under normal circumstances, it can be troublesome in certain environments when a proxy is in use or when an Express Route circuit is used with certain configurations.  In these scenarios, an alternative host name is needed to allow for proper routing to the Event Hubs service.
```